src/puptoo/process/profile.py

```diff
@@ -225,6 +225,7 @@ def system_profile(
 
     if rpm_ostree_status:
         profile["host_type"] = "edge"
```

Comment:
@CodeHeeler - does that mean each time when host is using RHEL for edge, rpm_ostree_status will be present?

Reply:
@kgaikwad I'm not sure how we determine edge hosts and unfortunately do not have the bandwidth to dig into it right this minute.  @xiangce do you know how Insights determines which hosts are edge? Thanks in advance.

Reply:
@kgaikwad - A new combiner `RhelForEdge` was added to the insights-core to check if the host an RHEL for Edge. I suggest replacing the `rpm_ostree_status` with the new combiner here in the puptoo.

- https://github.com/RedHatInsights/insights-core/blob/3.0/insights/combiners/rhel_for_edge.py#L21

and for the context why we should not only rely on the `RpmOstreeStatus` parsers, please refer to https://issues.redhat.com/browse/RHELPLAN-141835

Reply:
Also, does that mean `rpm_ostree_status`  will let us know whether the system is edge computing but not its OS?

Reply:
@r14chandra - I didn't check that here the content of the parser is also required. So, the current logic looks good, it's also the same as the `RhelForEdge` combiner, this combiner also checks the `RpmOstreeStatus` in the first priority and turns to other parsers when the `RpmOstreeStatus` is not available.

Regarding the archive for `RpmOstreeStatus` (also for the RhelForEdge), I will share it with you via slack soon, after I got one.

Reply:
> Also, does that mean `rpm_ostree_status` will let us know whether the system is edge computing but not its OS?

Technically, only checking if the  `rpm_ostree_status` get triggered is not so accurate/sufficient, since the `"RHCOS"` (core OS) also use the `"rpm-ostree"` so the `RhelForEdge` combiner also checks the following [items](https://github.com/RedHatInsights/insights-core/blob/c03218480ae0d0c1d89891ca419f846214507184/insights/combiners/rhel_for_edge.py#L66-L70):
```
        if rpmostreestatus:
            origin = rpmostreestatus.query.deployments.origin
            origin_check = [item.value.endswith("edge") for item in origin]
            if origin_check and all(origin_check):
                self.is_edge = True
```
Only in Edge computing systems, the above `origin` contains items ending with `"edge"`.

And FYI, there is a plan to make the identification of `Edge` more accurate, for details please refer to RHELPLAN-141835 and THEEDGE-3043. We'll update the `RhelForEdge` combiner once that is finished.

So, in short, for the safe, for now, I think we may also need to update the identifying logic here.

Reply:
@xiangce If I add this check, then the tests fail for [test_host_type](https://github.com/RedHatInsights/insights-puptoo/blob/master/tests/test_host_type.py#L85) because `origin` doesn't contain `edge`. 

So, IIUC, [DATA_1](https://github.com/RedHatInsights/insights-puptoo/blob/master/tests/test_host_type.py#L49) is of none edge system. So, may I delete this from the test? Or am I missing anything?
@kgaikwad

Reply:
@r14chandra - sounds good to me. Alternatively, we can update the DATA_1 according to the sample archive. I think both work the same.

Reply:
@xiangce, @r14chandra  - In my opinion, can we keep this data DATA_1 as it is and fix the assert statement? That will cover both cases.